uber/config.py

```diff
@@ -363,6 +363,8 @@ def _is_intstr(s):
 
 c.TABLE_PRICES = defaultdict(lambda: _config['table_prices']['default_price'],
                              {int(k): v for k, v in _config['table_prices'].items() if k != 'default_price'})
```

Comment:
I guess I need to check my ``pep8`` config, since I'd have expected it to flag this with a style violation.  You're supposed to put spaces around operators, i.e. ``c.MAX_TABLES + 1`` instead of ``c.MAX_TABLES+1``.  Not a huge deal, but I'll try to remember and find the time to see why ``pep8`` didn't catch that in Travis.